sky/clouds/service_catalog/oci_catalog.py

```diff
@@ -179,10 +179,16 @@ def get_vcpus_mem_from_instance_type(
 
 def get_image_id_from_tag(tag: str, region: Optional[str]) -> Optional[str]:
     """Returns the image id from the tag."""
+    # Always try get region-specific imageid first (for backward compatible)
     image_str = common.get_image_id_from_tag_impl(_image_df, tag, region)
+    if image_str is None:
+        # Support cross-region (general) imageid
+        image_str = common.get_image_id_from_tag_impl(_image_df, tag, 'general')
```

Comment:
nit: Can we use `None` instead of `'general'`? It seems to be fine to leave the region field empty.